package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@chainsafe/libp2p-noise",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

```suggestion
  "version": "5.0.0",
```

This should be a major, the `libp2p-crypto` update pulled in by the `peer-id` update breaks compatibility with node 14 and below.

npm will happily install it, but the `engines` field in `libp2p-crypto` was changed to `>=15.0.0` which will make yarn sad if not running the correct node version.

---

Thanks for the review!
Good catch